### ISSUE TITLE
'Invalid flow specified' when adding integration to HA

### ISSUE THREAD

**Issue by vanderhavecn** [boshjosh1918/hacs-rako#1]
When i try adding the custom Rako integration with my Rako wireless hub (the latest one supporting cloud and Homekit) and i use my local hub ip, 'admin' username and the password I've configured in the web backend of the hub, I get an error. 

![Screenshot_20230321_192019_Home Assistant](https://user-images.githubusercontent.com/490328/226704983-54782799-a4dd-480a-9a64-454b9e2aed2f.jpg)
Invalid flow specified.

Anyone have an idea what I'm doing wrong here? Or does this integration not support the newer wireless hub?

**Comment by boshjosh1918** [comment]
Hi @vanderhavecn,

This repository is simply a copy of the code from the official Rako integration which I found from a pull request into the official home assistant core repository.

The pull request was incomplete and never merged. I simply got bored of waiting and so copied the code to this repository so I could easily try it out in HACS.

While I _was_ able to complete the initial setup and the room names from my Rako hub were loaded into home assistant as lights, they never actually controlled my setup and were permanently offline. My Rako setup uses a RakLink and wireless bridge. It is fairly old, so I don't know whether I have the same system as you.

Unfortunately I have never been able to find a Rako integration that properly works and as far as I am aware there are none out there. I know Rako bridges have some sort of HTTP API, however I have never been able to build anything myself.
